ports/nvtt/vcpkg.json

```
@@ -1,11 +1,11 @@
 {
   "name": "nvtt",
   "version": "2.1.2",
-  "port-version": 6,
+  "port-version": 7,
   "description": "Texture processing tools with support for Direct3D 10 and 11 formats.",
   "homepage": "https://github.com/castano/nvidia-texture-tools",
   "license": "MIT",
-  "supports": "!x86 & !(windows & (arm | uwp))",
+  "supports": "!x86 & !(windows & (arm | uwp)) & !(arm64 & osx)",
```

```suggestion
  "supports": "!x86 & !uwp & !(windows & arm) & !(osx & arm64)",
```

uwp implies Windows; I think this form is a bit clearer?

---

This version does not support arm at all, so `!x86 & !uwp & !arm` also works